docs/index.md

```diff
@@ -1401,66 +1436,43 @@ any special hypotheses to use. To do that:
+* Up: Move the selected statements up in the list.
+* Down: Move the selected statements down in the list.
 * "+": Add a statement (which you then type in).
 * Trash can: Delete the selected statement(s).
 * Duplicated "+": Copy the selected statement.
-* Merge: Merge the selected statements (they must be similar).
+* Merge: Merge the selected statement to a similar statement.
   Before clicking this button select only one statement;
   the other similar one will be detected by metamath-lamp.
 * Magnifying glass: Search for a statement pattern in the current context.
   The selected pattern one (if any) will be added as a new statement.
-  See below for more about search patterns.
-* A arrow: Apply a substitution to all selected statements.
+  See [search patterns](#search-patterns) for more about search patterns.
+* A with arrow: Apply a replacement (aka substitution) to all statements.
+  See [replacement](#replacement) for more information.
 * Network: Unify. If no statements are selected, it will attempt to unify
   all statements to create a proof. If a statement is selected, it will
-  open a dialogue to start a bottom-up search for a proof.
+  open a dialogue to start a bottom-up search for a proof; see
+  [proving bottom-up](#proving-bottom-up) for more about that.
 
-Under the editor command tab bar is basic information about the proof
-(such as its description).
-
-#### Basic information about the proof
-
-The basic information about the proof are the proof's description,
-variables, and disjoints. Click on the *section name* to edit this
-information. You can also select the editable fields, but
-description is odd - by default, you have to use Alt+left click to edit it,
-while just left click selects part of its text.
-
-Here's some about those fields:
-
-* Description: The description of the proof.
-  Note: This description is *not* currently generated as a comment to be
-  inserted into a database.
-* Variables: A list of variables.
-* Disjoints; A list of variables that are disjoint.
-
-<!--
-Omitted, since description is currently not generated as a comment:
-
-  in the final Metamath database just before the proof.
-  If you are following the conventions of `set.mm`, the first sentence
-  should be an English description of what is proved. Surround
-  Metamath statements with backquotes (so they can be typographically formatted)
-  and precede references to another with an isolated "~".
-  Conventionally this includes, at its end, a statement like
-  "(Contributed by NAME, DD-MMM-YYYY)" where DD-MMM-YYYY is the date
-  the proof was completed and MMM is the 3-letter English name
-  of the month.
--->
-
-Under the basic information about the proof
-are a list of statements in the proof.
+Under the editor command icon bar is the
+[fundamental proof information](#fundamental-proof-information)
+as we've already discussed.
+Below the editor command icon bar is the list of statements in the proof,
```

Comment:
Probably it will be more consistent if this

```
Below the editor command icon bar is the list of statements in the proof
```

is changed to 

```
Below the fundamental proof information is the list of statements in the proof
```

because the guide already states that `Under the editor command icon bar is the fundamental proof information`.